### ISSUE TITLE
Wrong color assignment accoriding to Pythia8

### ISSUE THREAD

**Issue by andriish** [LucianHL/SuperChic#136]
Pythia complains about the events produced with process 74:
```
<event>
11    0   1.    0.239724285E+03    0.729740000E-02    0.467969600E+00
             2212        1    0    0    0    0 -0.464761314E+00  0.237041741E+00  0.638506801E+04  0.638506810E+04  0.938272046E+00 0. 9.
             2212        1    0    0    0    0  0.484880129E-01 -0.523724334E+00 -0.635498034E+04  0.635498043E+04  0.938272046E+00 0. 9.
               93        2    0    0    0    0  0.416273301E+00  0.286682593E+00 -0.300876679E+02  0.259951464E+03  0.258203874E+03 0. 9.
          1000024        2    3    0    0    0  0.104726007E+02  0.391947753E+01  0.664190765E+02  0.120567539E+03  0.100000000E+03 0. 9.
         -1000024        2    3    0    0    0 -0.100563274E+02 -0.363279494E+01 -0.965067444E+02  0.139383925E+03  0.100000000E+03 0. 9.
          1000022        1    4    0    0    0  0.462112106E+01  0.650449685E+01  0.580304210E+02  0.991523726E+02  0.800000000E+02 0. 9.
                2        1    4    0  501    0 -0.560379794E-01  0.231601432E+01  0.124963652E+02  0.127092960E+02  0.124555472E-06 0. 9.
               -1        1    4    0    0  501  0.590751760E+01 -0.490103364E+01 -0.410770965E+01  0.870587006E+01  0.292001932E-06 0. 9.
          1000022        1    5    0    0    0 -0.172282680E+02  0.493710369E+01 -0.720472432E+02  0.109142079E+03  0.800000000E+02 0. 9.
               -2        1    5    0  501    0  0.108919271E+02 -0.776909171E+01 -0.218750019E+02  0.256419300E+02  0.461695593E-06 0. 9.
                1        1    5    0    0  501 -0.371998654E+01 -0.800806914E+00 -0.258449927E+01  0.459991609E+01  0.000000000E+00 0. 9.
```
 PYTHIA Error in ProcessLevel::checkColours: incorrect colour assignment  
 PYTHIA Abort from Pythia::next: processLevel failed; giving up

**Comment by andriish** [comment]
Hi @LucianHL, this is seen for the following processes as well: 
```
	327 - SHOWER_LHC13ppunw_LHC13ppunw_55_el_lhe (Failed)
	333 - SHOWER_LHC13ppunw_LHC13ppunw_56_el_lhe (Failed)
	339 - SHOWER_LHC13ppunw_LHC13ppunw_57_el_lhe (Failed)
	345 - SHOWER_LHC13ppunw_LHC13ppunw_58_el_lhe (Failed)
	369 - SHOWER_LHC13ppunw_LHC13ppunw_68_el_lhe (Failed)
	405 - SHOWER_LHC13ppunw_LHC13ppunw_74_el_lhe (Failed)
	441 - SHOWER_LHC13ppunw_LHC13ppunw_82_el_lhe (Failed)
	447 - SHOWER_LHC13ppunw_LHC13ppunw_83_el_lhe (Failed)
	453 - SHOWER_LHC13ppunw_LHC13ppunw_84_el_lhe (Failed)
	459 - SHOWER_LHC13ppunw_LHC13ppunw_54_sd_lhe (Failed)
	463 - SHOWER_LHC13ppunw_LHC13ppunw_54_dd_lhe (Failed)
	467 - SHOWER_LHC13ppunw_LHC13ppunw_55_sd_lhe (Failed)
	471 - SHOWER_LHC13ppunw_LHC13ppunw_55_dd_lhe (Failed)
	475 - SHOWER_LHC13ppunw_LHC13ppunw_56_sd_lhe (Failed)
	479 - SHOWER_LHC13ppunw_LHC13ppunw_56_dd_lhe (Failed)
	483 - SHOWER_LHC13ppunw_LHC13ppunw_57_sd_lhe (Failed)
	487 - SHOWER_LHC13ppunw_LHC13ppunw_57_dd_lhe (Failed)
	491 - SHOWER_LHC13ppunw_LHC13ppunw_58_sd_lhe (Failed)
	495 - SHOWER_LHC13ppunw_LHC13ppunw_58_dd_lhe (Failed)
	499 - SHOWER_LHC13ppunw_LHC13ppunw_68_sd_lhe (Failed)
	503 - SHOWER_LHC13ppunw_LHC13ppunw_68_dd_lhe (Failed)
```

**Comment by LucianHL** [comment]
I will look into this, although for sure e.g. processes 55-58 were working in previous iterations, so that is odd

**Comment by andriish** [comment]
Will be resolved with #197
